Arrowgene.Ddon.GameServer/Handler/MailSystemMailGetAllItemHandler.cs

```diff
@@ -0,0 +1,95 @@
+using Arrowgene.Ddon.GameServer.Dump;
+using Arrowgene.Ddon.Server;
+using Arrowgene.Ddon.Shared.Entity.PacketStructure;
+using Arrowgene.Logging;
+using Arrowgene.Ddon.Shared.Model;
+using Arrowgene.Ddon.Shared.Entity.Structure;
+using System.Linq;
+using System;
+
+namespace Arrowgene.Ddon.GameServer.Handler
+{
+    public class MailSystemMailGetAllItemHandler : GameRequestPacketHandler<C2SMailSystemMailGetAllItemReq, S2CMailSystemGetAllItemRes>
+    {
+        private static readonly ServerLogger Logger = LogProvider.Logger<ServerLogger>(typeof(MailSystemMailGetAllItemHandler));
+
+        public MailSystemMailGetAllItemHandler(DdonGameServer server) : base(server)
+        {
+        }
+
+        public override S2CMailSystemGetAllItemRes Handle(GameClient client, C2SMailSystemMailGetAllItemReq request)
+        {
+            var pcap = new S2CMailSystemGetAllItemRes.Serializer().Read(pcap_data);
+
+            var result = new S2CMailSystemGetAllItemRes()
+            {
+                MessageId = request.MessageId,
+            };
+            
+            bool toItemBag = false;
+            switch (request.StorageType)
+            {
+                case 13: // ItemPost   StorageType = 13
```

Comment:
Please add a TODO comment so whenever Item Post gets implemented it's easy to find this through Ctrl+F